src/colorize.cr

```diff
@@ -158,11 +158,13 @@ struct Colorize::Object(T)
   COLORS = %w(black red green yellow blue magenta cyan light_gray dark_gray light_red light_green light_yellow light_blue light_magenta light_cyan white)
   MODES  = %w(bold bright dim underline blink reverse hidden)
 
+  class_property? enabled : Bool = STDOUT.tty? && STDERR.tty?
```

Comment:
I think this class_property should belong to `module Colorize`. I found it misleading to be inside the generic class.